insightconnect_plugin_runtime/api/schemas.py

```diff
@@ -41,13 +55,35 @@ class ActionTriggerInputSchema(Schema):
     version = fields.Str(required=True)
 
 
+class TaskInputBodySchema(Schema):
+    action = fields.Str(required=True)
```

Comment:
Should this be `task` instead?
```suggestion
    task = fields.Str(required=True)
```